modules/social_features/social_landing_page/social_landing_page.module

```diff
@@ -55,19 +55,7 @@ function social_landing_page_theme() {
   $theme_templates['node__featured'] = [
     'base hook' => 'node',
   ];
-  $theme_templates['node__event__featured'] = [
-    'base hook' => 'node',
-  ];
-  $theme_templates['node__topic__featured'] = [
-    'base hook' => 'node',
-  ];
-  $theme_templates['node__page__featured'] = [
-    'base hook' => 'node',
-  ];
-  $theme_templates['node__book__featured'] = [
-    'base hook' => 'node',
-  ];
-  $theme_templates['node__landing_page__featured'] = [
+  $theme_templates['node__featured__sky'] = [
```

Comment:
We already declare template suggestions in a theme (--sky only be used when enabled).
But when want to use template from module we need to additionally declare it in `hook_theme()`.
So this just needs to be here, since we keep template files in the module.

Reply:
Hey Taras,

Good point 😋 I think the `--sky` template for this module should also be moved to the theme layer, out of the module.

The reasoning here is as follows:
- The module contains the minimum templates it needs to work (ideally, even outside of Open Social)
- The theme layer contains any changes that it needs to make to this such as branding and in this case the sky flavour.

Does that make sense or do you see reasons not to do this?

Reply:
I agree and moved all `--sky` templates to the socialblue theme.